src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/CreateVideoBookingRequest.kt

```diff
@@ -137,7 +137,7 @@ data class Appointment(
   @field:NotNull(message = "The date for the appointment is mandatory")
   @field:Future(message = "The date for the appointment must be in the future")
   @Schema(description = "The future date for which the appointment will start", example = "2022-12-23")
-  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "uuuu-MM-dd")
```

Comment:
This was the only way I could get it to accept the leniency configuration changes.

I struggled to find any real examples of why this would be.  `yyyy` represents an era date where as `uuuu`.  Given we are not recording BCE (before common era) dates this is fine.